server/datastore/mysql/migrations/tables/20231207133731_FixStatsTypes.go

```diff
@@ -0,0 +1,40 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231207133731, Down_20231207133731)
+}
+
+func Up_20231207133731(tx *sql.Tx) error {
+	// Updating some bad data from osquery (which will be ignored in a later PR).
```

Comment:
Will this be explained on the next PR? (Curious to know what's going on here.)

Reply:
I couldn't alter the table -- the merge was complaining about `last_executed=0000-00-00 00:00:00`. Maybe there was a way to force the column update and keep the `last_executed` values. In the next PR, I will not ingest scheduled query stats with executions == 0, so this should prevent any zero-value `last_executed` from being inserted into the table in the first place.

Reply:
Ah makes sense. Yes, makes no sense to ingest stats with `executions=0`.